src/linker/ref/Mono.Linker.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RootNamespace>Mono</RootNamespace>
+    <Version>0.2.0.0</Version>
+    <LangVersion>latest</LangVersion>
+    <AssemblyName>illink</AssemblyName>
+    <Description>IL Linker</Description>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <!-- Nuget issues a warning about missing <reference> items in the generated nuspec,
+        used for packages.config which we do not support. -->
+    <NoWarn>$(NoWarn);NU5131</NoWarn>
+    <!-- We need to create a package that includes the build output of the cecil projectreference below. -->
+    <IncludeBuildOutput>true</IncludeBuildOutput>
+    <BuildOutputTargetFolder>ref</BuildOutputTargetFolder>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Condition="'$(UseLocalCecil)' == 'false'" Include="Mono.Cecil" Version="$(MonoCecilVersion)" />
```

Comment:
This project should be for illink only so it's fine to use package only

Reply:
It doesn't make a huge difference, but I'd prefer to leave the switch here just for consistency with the implementation build if that's ok with you. (I would like to be able to build illink against the submodule locally, and in that case it'd be nice if the ref/impl built the same way and didn't require the package).